src/internal_events/route.rs

```diff
@@ -3,11 +3,12 @@ use vector_core::internal_event::InternalEvent;
 
 #[derive(Debug)]
 pub struct RouteEventDiscarded<'a> {
```

Comment:
If this event is related to multiple events, maybe the `Event` should be pluralized.

Reply:
Considering @lukesteensen 's comment [here](https://github.com/vectordotdev/vector/pull/11820#discussion_r827376907) I dropped this event.
Considering the event is now sent to `_else`, it's not discarded anymore.